src/super_gradients/common/abstractions/abstract_logger.py

```diff
@@ -1,29 +1,18 @@
 import os
 import logging
 import logging.config
+from typing import Union
 
 from super_gradients.common.auto_logging import AutoLoggerConfig
 
 
-# Controlling the default logging level via environment variable
-DEFAULT_LOGGING_LEVEL = os.environ.get("LOG_LEVEL", "INFO").upper()
-
-# Set the default level for all libraries - including 3rd party packages
-logging.basicConfig(level=DEFAULT_LOGGING_LEVEL)
-
-
-def get_logger(
-    logger_name: str, training_log_path=None, logs_dir_path=None, log_level=DEFAULT_LOGGING_LEVEL
-) -> logging.Logger:
-    config_dict = AutoLoggerConfig.generate_config_for_module_name(
-        module_name=logger_name, training_log_path=training_log_path, logs_dir_path=logs_dir_path, log_level=log_level
-    )
-    logging.config.dictConfig(config_dict)
+def get_logger(logger_name: str, log_level: Union[str, None] = None) -> logging.Logger:
+    AutoLoggerConfig.getInstance()
     logger: logging.Logger = logging.getLogger(logger_name)
```

Comment:
I honestly don't see any problem here. Log files are intended to be verbose and to avoid any ambiguity in them. If we use only the last part of the logger name, we may run into name clash issue. For instance, user has `utils.py` file of his own, in SG we may have `utils.py` file as well, and hydra also has `utils.py`

Now compare imaginary output using full names:
```bash
[2022-11-03 12:44:37] DEBUG - hydra.core.utils - Setting JobRuntime:name=UNKNOWN_NAME
[2022-11-03 12:44:38] WARNING - super_gradients.sanity_check.utils - Environment check unsuccessful - OS not supported
[2022-11-03 12:44:39] ERROR- customer_repo.train.utils - DATASET_PATH not found
```

To short names:
```bash
[2022-11-03 12:44:37] DEBUG - utils - Setting JobRuntime:name=UNKNOWN_NAME
[2022-11-03 12:44:38] WARNING - utils - Environment check unsuccessful - OS not supported
[2022-11-03 12:44:39] ERROR- utils - DATASET_PATH not found
```

To me, the latter option looks more cryptic and untraceable.

What we can do, is change the way we create the loggers in SG.
Currently there are `logger = get_logger(__name__)`. What we can use is `logger = get_logger("SG.Trainer") instead`. But I'm, not convinced we should do this at all as logs are not meant to be neither small nor human-readable. I'm happy to discuss this)

Reply:
There are also `%(filename)s` and `%(module)s` variables that we can use btw

Reply:
Example of file logging `%(asctime)s %(levelname)s - %(name)s - %(message)s`
```
[2022-11-07 11:15:15] DEBUG - hydra.core.utils - Setting JobRuntime:name=app
[2022-11-07 11:15:16] DEBUG - super_gradients.training.utils.sg_trainer_utils - "events.out.tfevents.1667812390.Bloodaxe-PC.22104.0" will not be deleted
[2022-11-07 11:15:58] INFO - super_gradients.training.utils.sg_trainer_utils - TRAINING PARAMETERS:
    - Mode:                         Single GPU
    - Number of GPUs:               1          (1 available on the machine)
    - Dataset size:                 100000     (len(train_set))
    - Batch size per GPU:           128        (batch_size)
    - Batch Accumulate:             1          (batch_accumulate)
    - Total batch size:             128        (num_gpus * batch_size)
    - Effective Batch size:         128        (num_gpus * batch_size * batch_accumulate)
    - Iterations per epoch:         781        (len(train_set) / total_batch_size)
    - Gradient updates per epoch:   781        (len(train_set) / effective_batch_size)

[2022-11-07 11:15:58] INFO - super_gradients.training.sg_trainer.sg_trainer - Started training for 400 epochs (0/399)
```

Example of console logging `%(asctime)s %(levelname)s - %(filename)s - %(message)s`
```
[2022-11-07 11:22:34]  WARNING - redirects.py - NOTE: Redirects are currently not supported in Windows or MacOs.
[2022-11-07 11:22:35]  WARNING - callbacks.py - Failed to import deci_lab_client
[2022-11-07 11:22:35]  WARNING - deci_platform_sg_logger.py - Failed to import deci_lab_client
C:\Anaconda3\envs\sg\lib\site-packages\_distutils_hack\__init__.py:33: UserWarning: Setuptools is replacing distutils.
```

Reply:
ok. let's use the filename